TelemetryProcessors/PreAggregatedMetricsTelemetryProcessor.ts

```diff
@@ -0,0 +1,53 @@
+import Contracts = require("../Declarations/Contracts");
+import AutoCollecPreAggregatedMetrics = require("../AutoCollection/PreAggregatedMetrics");
+import * as TelemetryType from "../Declarations/Contracts";
+import {
+    MetricDependencyDimensions,
+    MetricExceptionDimensions,
+    MetricRequestDimensions,
+    MetricTraceDimensions
+} from "../Declarations/Metrics/AggregatedMetricDimensions";
+import Context = require("../Library/Context");
+
+export function preAggregatedMetricsTelemetryProcessor(envelope: Contracts.EnvelopeTelemetry, context: Context): boolean {
+    // Increment rate counters
+    switch (envelope.data.baseType) {
```

Comment:
You need to add the `_MS.ProcessedByMetricExtractors` custom dimension to these to indicate to breeze to not aggregate.